### PR TITLE
Player auto-discovery option for music block.

### PR DESCRIPTION
**Pull request by rtxm**
As I use different music players for different purposes (*Spotify* for online music, *Audacious* or *Lollypop* for local music) I needed the music block to be able to auto-detect the player currently active.

So here 's my take on it.

### CODE REVIEW

**Comment by rtxm** [comment]
Thanks for you review. I'll fix that.

File: blocks.md [hunk 1]
```diff
@@ -397,7 +408,7 @@ Proprietary nvidia driver required.
 
 Creates a block which displays the Nvidia GPU utilization, temperature, used and total memory, fan speed, gpu clocks. You can set gpu label, that displayed by default.
 
-Clicking the left button on the icon changes the output of the label to the output of the gpu name. Same with memory: used/total.
```
[the remark this answers is not in the document]
Comment (rtxm) [reply blocks.md]: Are you sure you want to keep trailing white spaces? Or are you telling me I should make a different PR to fixe those?

[the remark this answers is not in the document]
Comment (atheriel) [reply blocks.md]: A separate PR is probably unnecessary, but a separate commit is what I had in mind.

**Comment by rtxm** [comment]
Cleaner version using an `Option` for the player attribute.

**Comment by atheriel** [comment]
Looks good to me.

**Pull request merged**